### PR TITLE
Improve the README

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,22 +146,22 @@ pip3 install --pre wrtc --no-binary wrtc
...
+Contributions of any sizes are welcome.
 
 ### Special thanks to
 
 - [Authors](https://github.com/node-webrtc/node-webrtc/blob/develop/AUTHORS) for [node-webrtc](https://github.com/node-webrtc/node-webrtc).
 
 ### License
 
-The `python-webrtc` licence is [BSD 3-Clause License](LICENSE.md).
+The `python-webrtc` is publicized under the [BSD 3-Clause License](LICENSE.md).
```
Comment (MarshalX) [inline README.md:167]:
```suggestion
The `python-webrtc` is published under the [BSD 3-Clause License](LICENSE.md).
```

?

File: README.md [hunk 2]
```diff
@@ -22,14 +22,13 @@
...
-specification. The changes were applied to make library more Pythonic and add useful API like programmatic audio and
+This project follows the [W3C specification](https://w3c.github.io/webrtc-pc/) with some modifications and additions to make it work better with Python applications, with useful APIs like programmatic audio and
 video.
 
 ## DISCLAIMER
 
-This project under development and doesn't redy for any serious use! In the current stage it's possible to establish
-connection and working with audio. The project has segfaults and sigbuse in large numbers ✨
+This project is still under development and isn't ready for any serious use! In the current stage it's possible to establish
+connection and work with audio with a large number of segfaults and sigbuses. ✨
```
Comment (MarshalX) [inline README.md:31]: could you merge 2 line into 1?

File: README.md [hunk 3]
```diff
@@ -22,14 +22,13 @@
 
 > Stop making unstandard pure implementations of WebRTC and let's use the native library!
 
-This project tries to be like [W3C](https://w3c.github.io/webrtc-pc/) specification, but there is some edits out of
-specification. The changes were applied to make library more Pythonic and add useful API like programmatic audio and
+This project follows the [W3C specification](https://w3c.github.io/webrtc-pc/) with some modifications and additions to make it work better with Python applications, with useful APIs like programmatic audio and
 video.
```
Comment (MarshalX) [inline README.md:26]: could you merge 2 lines into 1?

Comment (roj1512) [reply README.md]: Done.

**Review by MarshalX — APPROVED** [review]

**Pull request merged**